README.md

```diff
@@ -9,4 +9,16 @@ This is a simple app using Spring Boot as part of [Red Hat OpenShift Application
 
 ## OpenAPI (formerly known as Swagger) Support
 
-The app uses [Spring Fox](http://springfox.github.io/springfox/) to generate an [OpenAPI spec](https://www.openapis.org/). You can view the spec at /swagger.json or [Swagger UI](https://swagger.io/swagger-ui/) at /swagger-ui.html. 
\ No newline at end of file
+The app uses [Spring Fox](http://springfox.github.io/springfox/) to generate an [OpenAPI spec](https://www.openapis.org/). You can view the spec at /swagger.json or [Swagger UI](https://swagger.io/swagger-ui/) at /swagger-ui.html. 
+
+## Added plugins
+
```

Comment:
These links are good Haitham, build you should include the tl;dr for **usage** in the README. Do I run it with `mvn test`, `mvn install`, `mvn verify`? A special maven plugin? Assume the reader knows maven.